Lesson21/todo-app/sahra/script.js

```diff
@@ -0,0 +1,43 @@
+document
+  .getElementById("todoForm")
+  .addEventListener("submit", function (event) {
+    event.preventDefault();
+    addItem();
+  });
+
+  const todoInputElement = document.getElementById("todoInput");
+  const toDoListElement = document.getElementById("toDoList");
+  function addItem(){
+    if(todoInputElement.value === ''){
+      alert("There is nothing to do...");
+    } else {
+      const toDoListItemElement = document.createElement("li"); // Node
+      toDoListItemElement.textContent = todoInputElement.value;
+      toDoListElement.appendChild(toDoListItemElement);
+      let span = document.createElement("span");
+      span.innerHTML = "\u00d7";
+      toDoListItemElement.appendChild(span);
+    }
+    todoInputElement.value.trim() === '';
```

Comment:
This statement compares 'todoInputElement.value.trim()' to ''. You need to use a single equality sign instead:
```suggestion
    todoInputElement.value.trim() = '';
```